crates/holochain/src/conductor/conductor/tests.rs

```diff
@@ -743,7 +747,11 @@ async fn test_installation_fails_if_genesis_self_check_is_invalid() {
         panic!("this should have been an error")
     };
 
-    assert_matches!(err, ConductorApiError::ConductorError(ConductorError::GenesisFailed { errors }) if errors.len() == 1);
+    if let ConductorApiError::ConductorError(inner) = err {
+        assert_matches!(*inner, ConductorError::GenesisFailed { errors } if errors.len() == 1);
+    } else {
+        panic!("this should have been an error too");
```

Comment:
I would have assumed it doesn't match if the if condition is false and therefor panic. Is this not the case?

Reply:
@freesig i think i only changed this because clippy asked me to at some point...

Reply:
I think I'd prefer just adding an `#[allow]` in this case but that's just me